packages/docs/src/components/BasicNiivueCanvas.jsx

```diff
@@ -17,21 +26,48 @@ export const BasicNiivueCanvas = ({ images, nvOpts }) => {
   // initialise niivue instance on first mount
   React.useEffect(() => {
     niivue.current = new Niivue({logLevel: 'debug', ...nvOpts});
+>>>>>>> origin/main
```

Comment:
@Sanjai-Shaarugesh , this needs to be removed please. Same as the merge conflict comment earlier